MapboxNavigation/RouteVoiceController.swift

```diff
@@ -192,13 +192,9 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
      
      - parameter instruction: The instruction to read aloud.
      - parameter locale: The `Locale` used to create the voice read aloud the given instruction. If `nil` the `Locale.preferredLocalLanguageCountryCode` is used for creating the voice.
+     - parameter ignoreProgress: A `Bool` that indicates if the routeProgress is added to the instruction.
      */
-    open func speak(_ instruction: SpokenInstruction, with locale: Locale?) {
-        guard let routeProgress else {
-            assertionFailure("routeProgress should not be nil.")
-            return
-        }
-        
+    open func speak(_ instruction: SpokenInstruction, with locale: Locale?, ignoreProgress: Bool = false) {
```

Comment:
To be honest, I am not sure if we really need the ignoreProgress switch here? 

We would need to make route progress optional as well for `modifiedInstruction.attributedText(for: routeProgress.currentLegProgress)`.